servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -545,6 +545,31 @@ public final Completable concat(Completable next) {
         return new CompletableConcatWithCompletable(this, next);
     }
 
+    /**
+     * Once this {@link Completable} is terminated successfully, subscribe to each {@link Completable} in {@code nexts}
+     * in a sequential fashion after termination, and the final terminal signals is propagated to the returned
+     * {@link Completable}. Any error from this {@link Completable} or from {@code nexts} {@link Completable} are
+     * propagated to the returned {@link Completable}.
+     * <p>
+     * This method provide equivalent functionality as:
+     * <pre>{@code
+     *   Completable original = ...;
+     *   Completable[] nexts  = ...;
+     *   Completable result = original;
+     *   for (int i = 0; i < nexts.length; ++i) {
+     *       result = result.concat(nexts[i]);
+     *   }
+     *   return result;
+     * }</pre>
+     *
+     * @param nexts {@link Completable}s to subscribe after this {@link Completable} terminates successfully.
+     * @return A {@link Completable} that emits the terminal signal of {@code nexts} {@link Completable}s, after this
+     * {@link Completable} has terminated successfully.
+     */
+    public final Completable concat(Completable... nexts) {
```

Comment:
we have discussed modifying some operators to for a single parameter when varargs are used, however it is convenient to have the operator just take varargs bcz then the usage doesn't have to specialize the call site if it has a collection/array.